doc/source/quickstart.rst

```diff
@@ -8,58 +8,28 @@ The following quick start guide is for versions of CESM2 that have
 already been ported to the local target machine. CESM2 is built on the
 CIME (Common Infrastructure for Modeling Earth) framework.
 
-Please refer to the CIME `porting`_ documentation if CIME has not
+Please refer to the `CIME Porting Documentation <http://esmci.github.io/cime/users_guide/porting-cime.html>`_ if CIME has not
```

Comment:
This paragraph should be combined with the previous paragraph.

Reply:
fixed.